app/src/components/WithAdminPermissions/index.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from "react-redux";
+import { isLoggedIn, redirectToLogin, isAdmin } from "../../state/ducks/auth";
+import { WithAdminPermissions } from "./AdminPermissions";
```

Comment:
Should `WithPermissions` be modified to take a property instead of having a separate `WithAdminPermissions` component?